src/BrpProxy/Mappers/VerblijfplaatsMapper.cs

```diff
@@ -31,4 +31,23 @@ public static class VerblijfplaatsMapper
     {
         return verblijfplaats?.InOnderzoek?.AanduidingGegevensInOnderzoek == "089999" ? true : null;
     }
+
+    public static bool? MapVastgesteldVerblijftNietOpAdres(this GbaVerblijfplaatsBeperkt? verblijfplaats, AdresseringBeperkt adressering)
```

Comment:
De implementatie van deze twee methods is precies hetzelfde. Dit is code duplicatie en niet goed voor onderhoudbaarheid.
Dit kan worden opgelost mbv de Program to Interface principe. Introduceer twee interfaces, bijv. IBrpVerblijfplaatsInOnderzoek (we, ook RvIG niet, willen GBA niet meer in tekst/code gebruiken) met InOnderzoek property en IAdresseringAdres met Adresregel1, Adresregel2, Adresregel3 en Land property. Dan kunnen de class namen worden vervangen met de interface namen en kan één van de twee methods weg